app/controllers/application_controller/performance.rb

```diff
@@ -203,6 +203,7 @@ def perf_set_or_fix_dates(options)
     edate_daily = edate.hour < 23 ? edate - 1.day : edate
     options[:edate_daily] = edate_daily
 
+    options[:typ] = "Hourly" if options[:typ] == "Daily" && edate_daily < sdate_daily
```

Comment:
In theory yes.
In practice it doesn't happen due assumptions on how data is processed in regular metrics and metrics_rollups tables.
In any case, it's an additional protection with the expression "edate_daily < sdate_daily".